File: Python/fibonacci.py
def fibonacci(n):
    print(1)
    first, second = 0, 1
    for _ in range(n-1):
        next_val = first + second
        print(next)
        first, second = second, next_val


if __name__ == "__main__":
    i = int(input("Please enter a positive integer greater than 1: "))
    fibonacci(i)

File: C++/fibonacci.cpp
#include <iostream>
using namespace std;

int main() {
    int n, first, second, next;

    cout << "Please enter a positive integer greater than 1: ";
    cin >> n;

    cout << 1 << endl;
    first = 0;
    second = 1;
    for (int i = 0; i < n-1; i++) {
        next = first + second;
        cout << next << endl;
        first = second;
        second = next;
    }

    return 0;
}
